### PR TITLE
refactor(core): Decouple server started event from internal hooks (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
Follow-up to #10162

### CODE REVIEW

File: packages/cli/src/telemetry/telemetry-event-relay.service.ts [hunk 1]
```diff
@@ -420,4 +430,71 @@ export class TelemetryEventRelay {
...
+			license_tenant_id: config.getEnv('license.tenantId'),
+			binary_data_s3: isS3Available && isS3Selected && isS3Licensed,
+			multi_main_setup_enabled: config.getEnv('multiMainSetup.enabled'),
+		};
+
+		const firstWorkflow = await this.workflowRepository.findOne({
+			select: ['createdAt'],
+			order: { createdAt: 'ASC' },
+			where: {},
+		});
```
Comment (ivov) [inline packages/cli/src/telemetry/telemetry-event-relay.service.ts:490]: Initially tried to pass this as an event arg from `Server` but importing the workflow repository there leads to `ServiceNotFoundError: Service with "MaybeConstructable<DataSource>" identifier was not found in the container. Register it before usage via explicitly calling the "Container.set" function or using the "@Service()" decorator.`

File: packages/cli/test/unit/InternalHooks.test.ts
Comment (ivov) [inline packages/cli/test/unit/InternalHooks.test.ts:1]: I was thinking to bring this back later, else let me know if we should rewrite it now.

File: packages/cli/test/unit/InternalHooks.test.ts [hunk 2]
```diff
@@ -1,97 +0,0 @@
...
-		mock(),
-	);
-
-	beforeEach(() => jest.clearAllMocks());
-
-	it('Should be defined', () => {
-		expect(internalHooks).toBeDefined();
-	});
-
-	it('Should forward license plan name and tenant id to identify when provided', async () => {
```
Comment (netroy) [inline packages/cli/test/unit/InternalHooks.test.ts:52]: is there any value in porting this test over?

File: packages/cli/test/unit/InternalHooks.test.ts
Comment (netroy) [reply packages/cli/test/unit/InternalHooks.test.ts:1]: it's fine to do this later.

File: packages/cli/src/telemetry/telemetry-event-relay.service.ts (hunk 1 above)
Comment (netroy) [reply packages/cli/src/telemetry/telemetry-event-relay.service.ts:490]: Leaving this in Server meant that this query would have run even if telemetry was disabled. This file is the correct place for this query.

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.53.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.53.0)

**Pull request merged**